Source/Windows/Common/InvalidationTracker.cpp

```diff
@@ -63,6 +63,10 @@ void InvalidationTracker::InvalidateContainingSection(uint64_t Address, bool Fre
 }
 
 void InvalidationTracker::InvalidateAlignedInterval(uint64_t Address, uint64_t Size, bool Free) {
+  if (!Address) {
+    Size = std::numeric_limits<uint64_t>::max();
```

Comment:
This will overflow when adding `FEX_PAGE_SIZE` to it, causing no pages to be invalidated instead.

Reply:
Oh good catch, thanks.